
The following have been reloaded with a version change:
  1) gcc/10.2.0 => gcc/13.0.1

Traceback (most recent call last):
  File "/burg/ccce/users/eav2136/projects/interpolation/01_get_bands/c_df_dft_mp2.py", line 83, in <module>
    Lov[ki, kj] = out.reshape(-1, nocc, nvir)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "/burg/berkelbach/users/eav2136/builds/spack/develop/spack/var/spack/environments/pyscf/.spack-env/view/lib/python3.10/site-packages/h5py/_hl/dataset.py", line 1009, in __setitem__
    self.id.write(mspace, fspace, val, mtype, dxpl=self._dxpl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5d.pyx", line 280, in h5py.h5d.DatasetID.write
  File "h5py/_proxy.pyx", line 114, in h5py._proxy.dset_rw
OSError: [Errno 122] Can't synchronously write data (file write failed: time = Fri Dec 15 05:05:41 2023
, filename = '/burg/ccce/users/eav2136/projects/interpolation/01_get_bands/tmp/tmplzj13qsl', file descriptor = 7, errno = 122, error message = 'Disk quota exceeded', buf = 0xf0eae540, total write size = 236544, bytes this sub-write = 236544, bytes actually written = 18446744073709551615, offset = 0)
Exception ignored in: <function H5TmpFile.__del__ at 0x1555404e89d0>
Traceback (most recent call last):
  File "/burg/berkelbach/users/eav2136/builds/pyscf/projected-cvs/pyscf/pyscf/lib/misc.py", line 914, in __del__
  File "/burg/berkelbach/users/eav2136/builds/spack/develop/spack/var/spack/environments/pyscf/.spack-env/view/lib/python3.10/site-packages/h5py/_hl/files.py", line 586, in close
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 360, in h5py.h5f.FileID._close_open_objects
RuntimeError: Can't decrement id ref count (unable to close file, errno = 5, error message = 'Input/output error')
